elements/lisk-p2p/src/peer/inbound.ts

```diff
@@ -27,14 +26,16 @@ import { SCServerSocket } from 'socketcluster-server';
 
 export const EVENT_CLOSE_INBOUND = 'closeInbound';
 export const EVENT_INBOUND_SOCKET_ERROR = 'inboundSocketError';
-
+const DEFAULT_PING_INTERVAL = 500;
```

Comment:
We should ping less often by default. Maybe once every 1 minute? We don't want to allocate too many resources to pinging; it creates a lot of messages in the network.